build-farm/platform-specific-configurations/mac.sh

```diff
@@ -70,7 +70,7 @@ else
     if [ "${ARCHITECTURE}" == "x64" ]; then
       # We can only target 10.9 on intel macs
       export cxx_flags_bucket="${cxx_flags_bucket} -mmacosx-version-min=10.9"
-    elif [ "${ARCHITECTURE}" == "aarch64" ]; then
+    elif [[ "${MACHINEARCHITECTURE}" == "x64" ]] && [[ "${ARCHITECTURE}" == "aarch64" ]]; then
```

Comment:
Will this elif trigger given the if above?

Reply:
yes it will trigger on an x64 machine that is cross-compiling aarch64

Reply:
Ugh, sorry missed the variable name being different